src/Microsoft.Identity.Client/Cache/StorageJsonValues.cs

```diff
@@ -38,5 +38,7 @@ internal static class StorageJsonValues
         public const string CredentialTypeIdToken = "IdToken";
         public const string AccountRootKey = "Account";
         public const string CredentialTypeOther = "Other";
+
+        public const string AppMetadata = "appmetadata";
```

Comment:
nit: why this extra line?

Reply:
If I guess correctly, this would be the Root-level key that we are going to use in token serialization (which may or may not have been implemented yet - I did not read through this +2000 lines PR).

But if that is the case, I would suggest @bgavrilMS to consider using `AppMetadata` case to match the existing style. Later we can also officially document this word, in/after Mark's [internal proposal](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview/pullrequest/687?path=%2FUnifiedSchema%2FSchema.md&_a=files).

Reply:
yes, this is done. This constant is used to format the key, which is all lowercase and contains "appmetadata"